### ISSUE TITLE
Strife: Stalkers Play Startup Sound Repeatedly

### ISSUE THREAD

**Issue by GeoffLedak** [chocolate-doom/chocolate-doom#506]
In the original Strife, the Stalker enemies (spider looking things) would make an initial noise when triggered (only once), and then make a ticking noise constantly when active.

In Chocolate Doom and Strife Veteran Edition they are constantly playing their trigger sound over and over which gets pretty annoying.

**Comment by fabiangreffrath** [comment]
From src/strife/info.c
```
    {       /*MT_STALKER*/
        186,        //doomednum
        S_SPID_00,      //spawnstate
        80,     //spawnhealth
        S_SPID_03,      //seestate
        sfx_spisit,     //seesound
        8,      //reactiontime
        sfx_spdatk,     //attacksound
        S_SPID_24,      //painstate
        40,     //painchance
        sfx_spdatk,     //painsound
        S_SPID_09,      //meleestate
        S_NULL,     //missilestate
        S_NULL,     //crashstate
        S_SPID_25,      //deathstate
        S_NULL,     //xdeathstate
        sfx_spidth,     //deathsound
        16,     //speed
        31*FRACUNIT,        //radius
        25*FRACUNIT,        //height
        100,        //mass
        0,      //damage
        sfx_spisit,     //activesound
        MF_SOLID|MF_SHOOTABLE|MF_SPAWNCEILING|MF_NOGRAVITY|MF_DROPOFF
        |MF_NODIALOG|MF_NOBLOOD|MF_COUNTKILL,       //flags
        NULL,       //namepointer
    },
```
we can see that both "seesound" and "activesound" refer to the same sound, i.e. "sfx_spisit".

**Comment by fabiangreffrath** [comment]
I assume this is meant to be "sfx_spdwlk"?

**Comment by haleyjd** [comment]
I'll verify it against the disasm tonight.

**Comment by haleyjd** [comment]
Chocolate Strife's behavior matches the 1.2 binary:
```
dseg03:00091130 mobjinfo_t <186, 1009, 80, 1012, sfx_spisit,  8, sfx_spdatk, 1033, 40,\
dseg03:00091130             sfx_spdatk, 1018, 0, 0, 1034, 0, sfx_spidth, 16, 1F0000h, \
dseg03:00091130             190000h, 100, 0, sfx_spisit, \
dseg03:00091130             MF_SOLID or MF_SHOOTABLE or MF_SPAWNCEILING or MF_NOGRAVITY or MF_DROPOFF or MF_NODIALOG or MF_NOBLOOD or MF_COUNTKILL,\
dseg03:00091130             0>
```
I should probably state that the mobjinfo was converted directly with a utility written by Kaiser, so it's exceedingly unlikely to contain errors. If stalkers do behave differently in the DOS executable, then it must be due to a special case that is missing elsewhere.

**Comment by haleyjd** [comment]
Verified there's no difference in 1.31, just for extra thoroughness. Closing this issue.

**Comment by haleyjd** [comment]
Reopening because I still hear a difference even though I can't currently explain it...

**Comment by haleyjd** [comment]
I have pretty much eliminated everything except the low-level sound engine's behavior. I do not believe Chocolate Doom cuts off sounds in the same manner as vanilla did.

**Comment by haleyjd** [comment]
Closing, fixed by 5160ceb.
